### DISCUSSION TITLE
Allow tag moderators to propose aliases/merges for tags (with approval)

### DISCUSSION THREAD

**Discussion by andypiper** [forem/forem#15718] [Feature Requests]
I'm the moderator of a couple of tags on DEV, and I've noticed that occasionally typos creep in. I'm also aware that it is possible for tags to be aliased to one another (we did this for #twitterapi -> #twitter on DEV itself)

More recently I've noticed that specifically, there are tags on DEV for #rapberry (mis-type of #raspberry), and that should potentially be merged or aliased, and both potentially pointed to #raspberrypi. I'm not a moderator of any of these tags, and I can manually work with the admins on DEV via Discord chat to have this done, but it might be cool to have a way for trusted users and/or tag moderators be able to propose such management tasks for higher level approval and action.

Thoughts? It would warrant an approval / admin step to avoid mods or trusted users from inadvertently merging distinct topics.

**Comment by citizen428** [comment]
This seems like a good idea what do you say @michael-tharrington / @ellativity?

**Reply by michael-tharrington** [reply]
Love this idea!!

**Reply by ellativity** [reply]
Same! I feels intuitive for tag mods to be able to do this.

**Reply by JennieOcken** [reply]
I'm also super pro this idea.  Can you make it happen @citizen428 or should I route it into a different pod?

**Reply by citizen428** [reply]
@JennieOcken 

> It would warrant an approval / admin step to avoid mods or trusted users from inadvertently merging distinct topics

This makes me think the admin experience pod should at least be consulted, potentially even responsible.

**Reply by JennieOcken** [reply]
@Ridhwana Do you think this could fit in with our upcoming high impact, low effort work we will be doing?
